hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -1458,11 +1479,27 @@ bool MDMParser::getSignalStrength(NetStatus &status)
             _net.act = ACT_GSM;
         }
 
-        sendFormated("AT+CSQ\r\n");
-        if (RESP_OK == waitFinalResp(_cbCSQ, &_net, CSQ_TIMEOUT)) {
-            ok = true;
-            status = _net;
+        // AT command used to collect signal stregnth is different for R410M radio
+        if (_dev.dev == DEV_SARA_R410) {
+            sendFormated("AT+UCGED=5\r\n");
+            if (RESP_OK != waitFinalResp()) {
```

Comment:
Needs `UCGED_TIMEOUT` or else this will default to 10s.

Reply:
As per the u-blox's AT command manual, the response time is not given, so I let it to default.
Any other suggestions than asking Flint or using 10? :) 
![image](https://user-images.githubusercontent.com/48732259/75720098-80dee900-5c9b-11ea-8521-f2e6508fd81c.png)